FueledUtils/LoadingState.swift

```diff
@@ -38,6 +58,9 @@ public enum LoadingState<Error: Swift.Error> {
 }
 
 public extension Action {
+	///
+	/// Returns the current loading state for a given action.
+	///
 	public func loadingState() -> SignalProducer<LoadingState<Error>, NoError> {
```

Comment:
Curious about why we did not make this a computed var instead of function without parameters 🤔

Reply:
@notbenoit true, I can easily deprecate it and create a computed property though, Swift is smart enough not to make the code ambiguous in usage. What do you think?

Reply:
Agreed, let's deprecate it. It'll be more in sync with what `Action` exposes (`isExecuting`, `isEnabled`, `values`, etc...)

Reply:
@stephanecopin I think the documentation should be changed to something like that:
`A producer for Signals that will send the action’s current LoadingState.`

Reply:
@stephanecopin here too ^ :)